packages/connect-core/src/utils/transactions.ts

```diff
@@ -55,42 +16,17 @@ export async function createDirectTransactionForApp(
   methodSignature: string,
   params: any[]
 ): Promise<Transaction> {
-  if (!app) {
-    throw new Error(`Could not create transaction due to missing app artifact`)
-  }
+  const appInterface = app.interface()
```

Comment:
We refactor the logic to use Ethers interface directly instead of having the old logic to search for the `methodAbiFragment` manually.